Fine tune Network Boot provisioning (#4040)
theforeman/foreman-documentation: guides/common/modules/con_using-network-boot-to-provision-hosts.adoc

[id="using-network-boot-to-provision-hosts"]
= Using network boot to provision hosts

> You can provision hosts with {ProjectName} by booting hosts over a network when your system administrator configured provisioning infrastructure services, such as the DHCP and TFTP services, in your {Project} environment.

Author: That works, thank you!

```suggestion
After integrating provisioning infrastructure services with {Project}, you can provision hosts with {ProjectName} by booting hosts over a network.
```

[id="using-network-boot-to-provision-hosts"]
= Using network boot to provision hosts

You can provision hosts with {ProjectName} by booting hosts over a network when you can configure provisioning infrastructure services, such as the DHCP and TFTP services.
Once the host boots, {ProjectServer} or {SmartProxyServer} provides operating system installation content that the host downloads.

> When the operating system is installed, the host registers to {Project} and {Project} performs an initial configuration of the host.

Author: ```suggestion
After the operating system has been installed, the host registers to {Project} and {Project} performs an initial configuration of the host.
```
Does this help?
